notebooks/vertex_matching_engine/solutions/matching_engine_query.ipynb

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
<p>The next code cell deserves a longer explanation. Maybe something about what gRPC is, and what protocol buffer are, and how &amp; why we are going to use them in this lab may really help. Also a note on the general syntax of a <code>.proto</code> or at least a link to resource explaining that would be useful. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/230/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>